src/utils/typings.ts

```diff
@@ -0,0 +1,11 @@
+import { Ref } from 'vue-demi'
+import { SignalListenerHandler } from 'vega'
+import { Result } from 'vega-embed'
+
+export type VegaView = Result['view']
```

Comment:
You can import the view type directly from Vega, no?

Reply:
You're right, fixed.